### DISCUSSION TITLE
How to view the output of wezterm.log_info (or warn, error)?

### DISCUSSION THREAD

**Discussion by haphamdev** [wezterm/wezterm#6313] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

wezterm 20240203-110809-5046fc22

### Ask your question!

As the title said, I'm debugging my configuration. I would like to view the logs to see if things work. 
But I could not find that log file.
Where is it?

I started Wezterm as a GUI application on MacOS, not from a CLI command.

**Comment by haphamdev** [comment]
I had to open a tmp file and writing to that file. But that's not ideal.

**Comment by wez** [comment, marked as answer]
https://wezfurlong.org/wezterm/config/lua/keyassignment/ShowDebugOverlay.html
which is assigned to ctrl-shift-L by default

**Reply by haphamdev** [reply]
Thank you very much.
